### ISSUE TITLE
Problem while training: zero mAp.

### ISSUE THREAD

**Issue by Yemen-Romanian** [victordibia/handtracking#53]
Hello ,@victordibia, and thank you for your work! I have almost the same issue with training my custom hand detector as in Issue #5. I use a pre-trained model on EgoHands dataset you have provided and I do transfer learning on EgoFinger dataset. This dataset includes hand images taken from egocentric viewpoint. Image size is 640x480, format is png. For testing purposes I train on a little subset. But after every epoch I get zero mAp.
Also, I tried to learn by taking vanilla SSD and train on my dataset, but result is the same.
I am pretty sure that I have generated right csv files and tfrecords. On tensorboard "Image" tab I can view visualization of ground truth boxes and they are correct.
Are there any thoughts for what can be done? Thank you in advance!

**Comment by victordibia** [comment]
The Ego finger dataset looks interesting!
Zero mAP looks like an issue with your pipeline config or issues with incorrect labels.
Are you able to follow the TF Object detection API examples and successfully train a model?

**Comment by victordibia** [comment]
Hi @Yemen-Romanian ,
If you have any updates to share, please feel free to post here...
Closing for now
